### PR TITLE
Development Workflow: Explicitly state that contributors should not create multiple PRs for a single issue #6867

### LINKED ISSUE #6867: Development Workflow: Explicitly state that contributors should not create multiple PRs for a single issue
Contributors should not be opening multiple PRs for the same issue (e.g. Closing a PR and creating a new one). Doing this makes it difficult to keep track of past reviews/comments on the PR. We should consider stating this explicitly in our development workflow document.

### PR DESCRIPTION
**Pull request by PocketTesting**
Fixes #6867

**Outline of Solution**

I put more emphasis on not creating new PRs in the "Updating the PR" section of process.md.  However, I'm not sure if the full list of dos and don'ts is also supposed to be added to fix this issue.

### CODE REVIEW

File: docs/process.md [hunk 1]
```diff
@@ -179,8 +179,8 @@ Your code will be reviewed, in this sequence, by:
 
 #### Updating the PR
 
-If you are tasked to update your PR either by Travis CI or by your reviewer, there is no need to close the PR and open a new one.
-You will simply make and push the updates to the same branch used in the PR, essentially repeating [step 3](#step-3-fix-the-issue).
+If you are tasked to update your PR either by Travis CI or by your reviewer, **do not** close the PR and open a new one.
+You must make and push the updates to the same branch used in the PR, essentially repeating [step 3](#step-3-fix-the-issue).
```
Comment (whipermr5) [inline docs/process.md]: I prefer `should` instead of `must` as we're prescribing something here.

**Comment by whipermr5** [comment]
Hey @PocketTesting, welcome to TEAMMATES! Good job following the development process so far! A question: is @AdamDrennan your account too?

**Comment by PocketTesting** [comment]
Thanks, @whipermr5.  Yes, that's another account I made, and I must have accidentally logged into it when I made those commits.  Is that an issue, or should I just finish the requested change using this account?

**Comment by whipermr5** [comment]
@PocketTesting It's fine; I was just curious 😆

**Pull request merged**